packages/components/psammead-radio-schedule/src/ProgramCard/index.jsx

```diff
@@ -0,0 +1,244 @@
+import React from 'react';
+import styled from 'styled-components';
+import { mediaIcons } from '@bbc/psammead-assets/svgs';
+import { GEL_SPACING, GEL_SPACING_DBL } from '@bbc/gel-foundations/spacings';
+import {
+  C_EBON,
+  C_KINGFISHER,
+  C_METAL,
+  C_POSTBOX,
+  C_SHADOW,
+  C_WHITE,
+} from '@bbc/psammead-styles/colours';
+import {
+  getSansBold,
+  getSansRegular,
+  getSerifMedium,
+} from '@bbc/psammead-styles/font-styles';
+import {
+  getLongPrimer,
+  getMinion,
+  getPica,
+} from '@bbc/gel-foundations/typography';
+import { Link } from '@bbc/psammead-story-promo';
+import { oneOf, shape, string } from 'prop-types';
+import { scriptPropType } from '@bbc/gel-foundations/prop-types';
+import VisuallyHiddenText from '@bbc/psammead-visually-hidden-text';
+
+const CardWrapper = styled.div`
+  padding-top: ${GEL_SPACING};
+  background-color: ${C_WHITE};
+  position: relative;
+  border: 1px solid transparent;
```

Comment:
```suggestion
  border: 0.0625rem solid transparent;
```
Suggesting this for consistency - as through out psammead we tend to use `rem`